app/src/main/java/com/vci/vectorcamapp/settings/presentation/components/SettingsActionTile.kt

```diff
@@ -0,0 +1,76 @@
+package com.vci.vectorcamapp.settings.presentation.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import com.vci.vectorcamapp.R
+import com.vci.vectorcamapp.core.presentation.components.tile.ActionTile
+import com.vci.vectorcamapp.ui.extensions.colors
+import com.vci.vectorcamapp.ui.extensions.dimensions
+
+@Composable
+fun SettingsActionTile(
+    title: String,
+    modifier: Modifier = Modifier,
+    onClick: (() -> Unit),
+    content: (@Composable ColumnScope.() -> Unit)? = null,
+) {
+    val tileContent: @Composable () -> Unit = {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(MaterialTheme.dimensions.spacingMedium),
+            modifier = Modifier.padding(MaterialTheme.dimensions.paddingLarge)
+        ) {
+            Row(
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = title,
+                    style = MaterialTheme.typography.headlineSmall,
+                    color = MaterialTheme.colors.textPrimary
+                )
+
+                Box(
+                    contentAlignment = Alignment.Center,
+                    modifier = Modifier
+                        .size(MaterialTheme.dimensions.componentHeightSmall)
+                        .background(
+                            color = MaterialTheme.colors.iconBackground,
+                            shape = CircleShape
+                        )
+                ) {
+                    Icon(
+                        painter = painterResource(R.drawable.ic_arrow_right),
+                        contentDescription = "Navigate",
+                        tint = MaterialTheme.colors.icon,
+                        modifier = Modifier.size(MaterialTheme.dimensions.iconSizeMedium)
+                    )
+                }
+            }
+            if (content != null) {
+                content()
+            }
+        }
+    }
+
+    ActionTile(
+        onClick = onClick,
+        modifier = modifier,
+        content = tileContent
```

Comment:
You dont need to define a variable for tileContent. You can just do:

ActionTile(
        onClick = onClick,
        modifier = modifier
) {
     // Render whatever you want here
}